pages/common/pcapfix.md

```diff
@@ -0,0 +1,24 @@
+# pcapfix
+
+> Repair damaged or corrupted `pcap` and `pcapng` files.
+> More information: <https://f00l.de/pcapfix/>.
+
+- Repair a `pcap`/`pcapng` file:
+
+`pcapfix {{path/to/file.pcap}}`
+
+- Repair an entire `pcap`/`pcapng` file:
+
+`pcapfix --deep-scan {{path/to/file.pcap}}`
+
+- Repair a `pcap`/`pcapng`` file and write the repaired file to a specified file:
```

Comment:
```suggestion
- Repair a `pcap`/`pcapng` file and write the repaired file to the specified location:
```

Reply:
Thanks for the hint! Now the sentence clearly looks better.